package.json

```diff
@@ -56,8 +62,6 @@
         "decimal.js": "10.1.1",
         "dompurify": "1.0.11",
         "eslint": "^7.32.0",
-        "eslint-plugin-es": "^4.1.0",
-        "eslint-plugin-jsdoc": "^36.1.0",
```

Comment:
**question:** Why is it removed? This is still in use and it does not look to be installed by peers. Removing this might add back IDE complains and will prevent it from working properly...

Reply:
Let continue discussion here https://github.com/oat-sa/tao-test-runner-qti-fe/pull/457/files#r774527114